samples/listener_sample/event_create_new_file.bal

```diff
@@ -0,0 +1,89 @@
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerinax/googleapis_drive as drive;
+import ballerinax/googleapis_drive.'listener as listen;
+
+configurable string callbackURL = ?;
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshUrl = drive:REFRESH_URL;
+configurable string refreshToken = ?;
+
+string fileName = "<NEW_FILE_NAME>";
+
+# Event Trigger class  
+public class EventTrigger {
+    
+    public function onNewFolderCreatedEvent(string folderId) {}
+
+    public function onFolderDeletedEvent(string folderID) {}
+
+    public function onNewFileCreatedEvent(string fileId) {
+        log:print("New File was created:" + fileId);
+    }
+
+    public function onFileDeletedEvent(string fileId) {}
+
+    public function onNewFileCreatedInSpecificFolderEvent(string fileId) {}
+
+    public function onNewFolderCreatedInSpecificFolderEvent(string folderId) {}
+
+    public function onFolderDeletedInSpecificFolderEvent(string folderId) {}
+
+    public function onFileDeletedInSpecificFolderEvent(string fileId) {}
+
+    public function onFileUpdateEvent(string fileId) {}
+}
+
+    drive:Configuration config = {
+        clientConfig: {
+            clientId: clientId,
+            clientSecret: clientSecret,
+            refreshUrl: refreshUrl,
+            refreshToken: refreshToken
+        }
+    };
+
+    listen:ListenerConfiguration configuration = {
+        port: 9090,
+        callbackURL: callbackURL,
+        clientConfiguration: config,
+        eventService: new EventTrigger()
+    };
+
+    listener listen:DriveEventListener gDrivelistener = new (configuration);
+
+    service / on gDrivelistener {
+        resource function post gsheet(http:Caller caller, http:Request request) returns string|error? {
+            error? procesOutput = gDrivelistener.findEventType(caller, request);
+            http:Response response = new;
+            var result = caller->respond(response);
+            if (result is error) {
+                log:printError("Error in responding ", err = result);
+            }
+        }
+    }
+
+public function main() returns error? {
+    drive:Client driveClient = check new (config);
+    drive:File|error response = driveClient->createFile(fileName);
+    if (response is drive:File) {
+        log:print(response.toString());
+    }
+}
```

Comment:
Add a new line in the end of a file.
Please fix in all the places

Reply:
Thanks @LakshanSS 
Fixed in all places